### DISCUSSION TITLE
FirebaseAuth.Initialize crashes on flutter chrome extension

### DISCUSSION THREAD

**Discussion by jay-athelas** [firebase/flutterfire#12975] [Q&A]
Hi Guys, is there a guide in how to implement firebase authentication for a flutter chrome extension App, I tried following available tutorials in official flutter fire and firebase docs. But the app crashes with White screen when compiled to a chrome extension.

**Comment by jay-athelas** [comment]
This is a ghost town

**Comment by Andrushens** [comment]
I faced the same issue and looks like found the solution - to fix it we can download firebase .js files and import them in CE app.
Referring to https://stackoverflow.com/questions/74822985/how-to-use-flutterfire-in-chrome-extension

Looks like it is CE's security problems that don't allow importing external scripts and downloading these files is the only way to make CE work with firebase
